README.md

```diff
@@ -30,30 +30,58 @@ testTools(gulp, {
 
 ```
 
-Running `gulp help` will now show all the tasks and a description (if provided) for each of the tasks.
+Running `gulp help` will show all the tasks and a description (if provided) for each of the tasks.
 
 
 ### Options
+ - es6: Specifies that your code is written in ES6 (+ JSX). You must also name your gulpfile `gulpfile.babel.js`. Defaults to `false`.
```

Comment:
@DullReferenceException It doesn't force JSX, it just includes the linting rules so that you don't have to enable both. It's actually more of a [godaddy-style](https://github.com/godaddy/javascript) thing than anything this tool changes.